lib/environment_config_transmogrifier.rb

```diff
@@ -50,12 +52,21 @@ def self.transmogrify(base_config, environment_templates, secrets: nil)
   end
 
   def self.processMustacheTemplate(value, input_hash, key)
+    unless @@memoize_mustache[value].nil?
+      unless @@memoize_mustache[value][input_hash].nil?
+        return @@memoize_mustache[value][input_hash]
+      end
+    end
+
```

Comment:
A more ruby-like way to write lines 55-59:

val = @@memoize_mustache.fetch(value, {})[input_hash]
return val if val

No need to check `val` for `nil`ness because it should be a non-nil non-false yaml object.